source/OptimaJet.DWKit.StarterApplication/Models/ProductActions.cs

```diff
@@ -6,7 +6,22 @@ namespace OptimaJet.DWKit.StarterApplication.Models
 {
     public class ProductActions : Identifiable<Guid>
     {
-        [Attr("types")]
-        public List<string> Types { get; set; }
+        [Attr("actions")]
+        public List<string> Actions { get; set; }
+    }
+
+    public class ProductActionRuns : Identifiable<Guid>
+    {
+        [Attr("action")]
+        public string Action { get; set; }
+
+        [Attr("products")]
+        public List<string> Products { get; set; }
+    }
+
+    public class ProductActionProjects : Identifiable<Guid>
```

Comment:
Maybe: `ProjectIdList`?
idk, ProductActionProjects confuses me :shrug:

Reply:
Changed to ProductActionProjectIds